libs/blocks/share/share.css

```
@@ -146,3 +147,13 @@ main > .section.center .share.inline p.icon-container {
   margin: 0 auto;
   justify-content: center;
 }
+
+@media screen and (min-width: 600px) {
+  .share .copy-to-clipboard::before {
+    left:50%;
+  }
+
+  .share .copy-to-clipboard::after {
+    left:50%;
+  }
+}
```

Would be good to clean up spacing here, a space after the colons and newline at end of file
```suggestion
@media screen and (min-width: 600px) {
  .share .copy-to-clipboard::before {
    left: 50%;
  }

  .share .copy-to-clipboard::after {
    left: 50%;
  }
}

```